wandb/__init__.pyi

```diff
@@ -12,7 +12,7 @@ For reference documentation, see https://docs.wandb.com/ref/python.
 from __future__ import annotations
 
 __all__ = (
-    "__version__",
+    "__version__", # doc:exclude
```

Comment:
This should go into `wandb/__init__.template.pyi` - this file is auto-generated from it.